docs/reference/query-dsl/query-string-query.asciidoc

```diff
@@ -63,6 +63,8 @@ with default operator of `AND`, the same query is translated to
 
 |`analyzer` |The analyzer name used to analyze the query string.
 
+|`quote_analyzer` |The analyzer name used to analyze the quote part of query string.
```

Comment:
So, I confirmed that this parameter forces the analyzer on every quoted part or the search string. I think its important to mention that it is optional, and that it overwrites any other analyzer for these quoted parts. Maybe also add that in this it is related to the [`search_quote_analyzer`](https://www.elastic.co/guide/en/elasticsearch/reference/master/analyzer.html#search-quote-analyzer) setting.